tuist/tuist
# `XPC` product can embed dynamic libraries and frameworks

Resolves: No issue was created, but it is impossible to embed dynamic libraries to the XPC, while it is should be alloed 

### Short description 📝

> Describe here the purpose of your PR.

This Pull request allows having dynamic libraries in the `XPC` services.
`XPC` was introduced in https://github.com/tuist/tuist/pull/5077 but for some reason dynamic libraries weren't added as a valid option for XPC https://github.com/tuist/tuist/pull/5077/files#diff-bb33503408b0fd321d9ca4ff9ddd6bb15edcb6bbcbf630daad76711a01169acaR449-R452

### How to test the changes locally 🧐
Run tuist generate for macos_app_with_xpc project

> Include a set of steps for the reviewer to test the changes locally (see [the documentation](https://docs.tuist.io/contributors/testing-strategy) for reference).

One should expect that dynamic libraries are embedded inside XPC

![image](https://github.com/tuist/tuist/assets/119268/af7a67fc-6145-41e8-9393-74711e6ff6eb)

### Contributor checklist ✅


### Reviewer checklist ✅
